app/parse.py

```diff
@@ -8,5 +14,28 @@
     duration: str
 
 
+def parse_single_course(soup: Tag) -> Course:
+    return (Course(
+        name=soup.select_one("a.ProfessionCard_title__fEWio > h3").text,
+        short_description=soup.select_one(
+            ".ProfessionCard_professionTags__2iarD + "
+            "p.typography_landingTextMain__Rc8BD"
+        ).text,
+        duration=soup.select_one(
+            ".ProfessionCardTags_regularTag__dqOGj:last-child span"
+        ).text
```

Comment:
Verify that the CSS selectors used here match the actual HTML structure of the webpage. If the selectors do not match, `soup.select_one` will return `None`, leading to an AttributeError when attempting to access `.text`.